src/databricks/labs/remorph/cli.py

```diff
@@ -34,8 +34,8 @@ def raise_validation_exception(msg: str) -> Exception:
 @remorph.command
 def transpile(
     w: WorkspaceClient,
-    source: str,
-    input_sql: str,
+    source_dialect: str,
+    input_folder: str,
```

Comment:
It should be **input_source** as in labs.yml

Reply:
Thanks, done!